grpc/pb-marshalling.go

```diff
@@ -428,6 +429,44 @@ func PBToCert(pb *corepb.Certificate) (core.Certificate, error) {
 	}, nil
 }
 
+func CertStatusToPB(certStatus core.CertificateStatus) *corepb.CertificateStatus {
+	ocspLastUpdatedNano := certStatus.OCSPLastUpdated.UnixNano()
+	revokedDateNano := certStatus.RevokedDate.UnixNano()
+	lastExpirationNagSentNano := certStatus.LastExpirationNagSent.UnixNano()
+	notAfterNano := certStatus.NotAfter.UnixNano()
+	reason := int64(certStatus.RevokedReason)
+	status := string(certStatus.Status)
+
+	return &corepb.CertificateStatus{
+		Serial:                &certStatus.Serial,
+		Status:                &status,
+		OcspLastUpdated:       &ocspLastUpdatedNano,
+		RevokedDate:           &revokedDateNano,
+		RevokedReason:         &reason,
+		LastExpirationNagSent: &lastExpirationNagSentNano,
+		OcspResponse:          certStatus.OCSPResponse,
+		NotAfter:              &notAfterNano,
+		IsExpired:             &certStatus.IsExpired,
+	}
+}
+
+func PBToCertStatus(pb *corepb.CertificateStatus) (core.CertificateStatus, error) {
+	if pb == nil || pb.Serial == nil || pb.Status == nil || pb.OcspLastUpdated == nil || pb.RevokedDate == nil || pb.RevokedReason == nil || pb.LastExpirationNagSent == nil || pb.OcspResponse == nil || pb.NotAfter == nil || pb.IsExpired == nil {
```

Comment:
I think we've been somewhat inconsistent about this, but I the general trend is to keep these checks in the grpc wrapper methods, rather than the PB conversion functions.

Reply:
Yeah, i was choosing to copy what `PBToCert` does, but can do it the other way instead.